file/jsonops.go

```diff
@@ -62,7 +63,22 @@ func (j *JSONOps) pullRawFile(filename string) ([]byte, error) {
 	}
 	defer jFile.Close()
 
-	return ioutil.ReadAll(jFile)
+	return io.ReadAll(jFile)
+}
+
+// WriteJSON writes a serialized JSON object or array to a file.
+func (j *JSONOps) WriteJSON(m interface{}, filename string) error {
```

Comment:
This is a duplicate of write obj. Use the existing function

Reply:
Updated PR